### PR TITLE
[bug 1170634] Fix issues in Travis lint build

### PR DESCRIPTION
**Pull request by rehandalal**
Just testing for now...

### CODE REVIEW

**Comment by rehandalal** [comment]
@mythmon: r?

File: scripts/travis/dispatch.sh [hunk 1]
```diff
@@ -5,10 +5,10 @@ SUITE=${1:-all}
 case $SUITE in
     all )
         scripts/travis/test.sh
-        scripts/lint.sh
+        pre-commit run --all-files
         ;;
     lint )
-        pre-commit run --all-files
+        pre-commit run --all-files --
```
Comment (mythmon) [inline scripts/travis/dispatch.sh]: What's the `--` do? And why isn't it on the other version on line 8?

Comment (rehandalal) [inline scripts/travis/dispatch.sh]: was a typo. already fixed.

**Comment by mythmon** [comment]
Nice! Thank you for tracking this down. I'm also glad we weren't the only ones with this problem.

**Comment by mythmon** [comment]
Travis isn't done yet, but the lint build did pass, so I'm going to merge this now.

**Pull request merged**